docs/resource-stacks.md

```diff
@@ -0,0 +1,46 @@
+# Resource Stacks 📚
+
+Machine Learning projects often vary in their size, from small-scale experimentation to large deployments, meaning that the infrastructure requirements also change and scale. For example, the infrastructure stack needed for deploying an LLM may require a GPU or vector database, which aren't usually needed in more general machine learning use-cases.
+
+Matcha accommodates both of these requirements, and currently offers two infrastructure stacks which we'll discuss in more detail here and show how you can get started with either.
+
+> Note: These stacks must be set before provisioning any resources and cannot currently be change whilst a deployed.
+
+## Available stacks
+
+### DEFAULT
+
+The `DEFAULT` stack. This stack is ideal for generic machine learning training and deployments and a good starting point. It includes:
+   * [Azure Kubernetes Service](https://azure.microsoft.com/en-gb/products/kubernetes-service)
+   * [ZenML](https://www.zenml.io/home)
+   * [Seldon Core](https://www.seldon.io/solutions/open-source-projects/core) (deployment)
+   * [MLflow](https://mlflow.org/) (experiment tracking)
+   * [Data Version Control](https://dvc.org/) storage
```

Comment:
Don't think we need the link to dvc.org here. The data version control storage is just storage, nothing is installed there.